content/en/news/releases/1.5.x/announcing-1.5/change-notes/index.md

```diff
@@ -4,4 +4,80 @@ description: Istio 1.5 release notes.
+- **Improved** `EnvoyFilter` matching performance #19786.
+- **Added** support for `HTTP_PROXY` protocol #19919.
+- **Improved** `iptables` setup to use `iptables-restore` by default #18847.
+- **Improved** Gateway performance by filtering unused clusters. This setting is disabled by default #20124.
+
+## Security
+
+- **Added** Beta authentication API. The new API separates peer (i.e mutual TLS) and origin (JWT) authentication into `PeerAuthentication` and `RequestAuthentication` respectively. Both new APIs are workload-oriented, as opposed to service-oriented in alpha `AuthenticationPolicy`.
+- **Added** [deny semantics](https://github.com/istio/api/blob/master/security/v1beta1/authorization.proto#L28) to Authorization Policy
+- **Graduated** [auto mutual TLS](/docs/tasks/security/authentication/auto-mtls/) from alpha to beta. This feature is now enabled by default.
+- **Improved** node level citadel agent by merging with pod level Istio agent. This improves the security posture by removing the requirement of a pod security policy.
+- **Improved** Istio by including certificate provisioning functionality within istiod.
+- **Improved** Support Kubernetes [`first-party-jwt`](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#service-account-tokens) as a fallback token for CSR authentication in clusters that [`third-party-jwt`](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection) is not supported.
+- **Added** Support Istio CA and Kubernetes CA to provision certificates for the control plane.
+- **Added** Istio Agent provisions a key and certificates for Prometheus.
+- **Graduated** SDS to stable and enabled by default. It provides identity provisioning for Istio Envoy proxies.
+
+## Telemetry
+
+- **Added** TCP protocol support for v2 telemetry.
+- **Added** gRPC response status code support in metrics/logs.
+- **Added** support for Istio Canonical Service.
+- **Improved** stability of v2 telemetry pipeline.
+- **Added** alpha-level support for configurability in v2 telemetry.
+- **Added** support for populating AWS platform metadata in Envoy node metadata.
+- **Improved** Stackdriver adapter for Mixer to support configurable flush intervals for tracing data.
+- **Added** support for a headless collector service to the Jaeger addon.
+- **Fixed** `kubernetesenv` adapter to provide proper support for pods that contain a dot in their name.
+- **Improved** the Fluentd adapter for Mixer to provide millisecond-resolution in exported timestamps.
+
+
+## Configuration management
+
+## Operator
+
+- **Replaced** the alpha `IstioControlPlane` API with the new [`IstioOperator`](/docs/reference/config/istio.operator.v1alpha1/) API to align with existing MeshConfig API.
+- **Added** `istioctl operator init` and `istioctl operator remove` commands.
+- **Improved** reconciliation speed with caching [operator#667](https://github.com/istio/operator/pull/667).
+
```

Comment:
@fpesce The link for 'reconciliation speed with caching should be: https://github.com/istio/istio/pull/20720 instead